python/GafferUI/ColorChooser.py

```diff
@@ -611,6 +614,31 @@ def setErrored( self, errored ) :
 	def getErrored( self ) :
 		return any( w.getErrored() for w in self.__numericWidgets.values() )
 
+	def setVisibleComponents( self, components ) :
+
+		self.__setVisibleComponentsInternal( components )
+
+	def getVisibleComponents( self ) :
+
+		return "".join( [ k for k, v in self.__sliders.items() if v.getVisible() ] )
+
+	def setStaticComponent( self, component ) :
+
+		self.__setStaticComponentInternal( component )
```

Comment:
Do we need a separate internal function? Could the implementation just move here?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/6003/commits/591819a5ff5d6ee70dfb7a4297375321b837fa0d